WalletWasabi.Gui/Global.cs

```diff
@@ -179,16 +177,16 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 					RegisterFeeRateProviders();
 					RegisterCoinJoinComponents();
 
-                    SystemAwakeChecker? systemAwakeChecker = await SystemAwakeChecker.CreateAsync(HostedServices.Get<CoinJoinManager>()).ConfigureAwait(false);
+					SleepInhibitor? sleepInhibitor = await SleepInhibitor.CreateAsync(HostedServices.Get<CoinJoinManager>()).ConfigureAwait(false);
 
-                    if (systemAwakeChecker is not null)
-                    {
-                        HostedServices.Register<SystemAwakeChecker>(systemAwakeChecker, "System Awake Checker");
-                    }
-                    else
-                    {
-                        Logger.LogInfo("System Awake Checker is not available on this platform.");
-                    }
+					if (sleepInhibitor is not null)
+					{
+						HostedServices.Register<SleepInhibitor>(sleepInhibitor, "Sleep Inhibitor");
+					}
+					else
+					{
+						Logger.LogInfo("System Awake Checker is not available on this platform.");
```

Comment:
Old name

Reply:
Please try to find `System Awake Checker` in your IDE whether it's correct everywhere

Reply:
Nice catch!